docs/requirements.txt

```diff
@@ -8,13 +8,16 @@ sphinx-markdown-tables
 sphinxcontrib-napoleon
 
 # Markdown cross-reference solver library
-git+https://github.com/SymbiFlow/sphinxcontrib-markdown-symlinks.git#egg=markdown_code_symlinks
+git+https://github.com/SymbiFlow/sphinxcontrib-markdown-symlinks.git@d912c75#egg=markdown_code_symlinks
```

Comment:
Is this needed now?

Reply:
Same as above

Reply:
Removed explicit revision specification.